doc/manuals/admin/database_model.md

```diff
@@ -301,7 +306,7 @@ Fields:
     the subscription.   
 -   **format**: the format to use to send notification, possible values are **JSON**
```

Comment:
What about "legacy" ...
That's also valid, right?

Reply:
No...  `"attrsFormat": "legacy"` uses "JSON", not "legacy". 

Why? I'm not fully sure, but I think this comes from the XML times, where `format` could be "XML" or "JSON". The "XML" alternative disappeared in Orion 1.0.0, leaving "JSON". So, new format used "keyValues", etc. leaving old "JSON" for NGSIv1 format (i.e. legacy notifications).

Reply:
NTC